include/networkit/networkit_export.h

```diff
@@ -0,0 +1,48 @@
+

```

Comment:
The naming convention of this file does not match NetworKit's. Maybe `Macros.hpp` would be a better name?